test/test/util/util.js

```diff
@@ -448,13 +448,13 @@ shaka.test.Util = class {
     const buffer = await shaka.test.Util.fetch(
         '/screenshot/diff' + parentUrlParams + paramsString);
     const json = shaka.util.StringUtils.fromUTF8(buffer);
-    const pixelsChanged = /** @type {number} */(JSON.parse(json));
+    const similarity = /** @type {number} */(JSON.parse(json));
 
-    // If the change threshold is exceeded, you can review the new screenshot
+    // If the minimum similarity is not met, you can review the new screenshot
     // and the diff image in the screenshots folder.  Look for images that end
     // with "-new" and "-diff".  If cropping doesn't work right, you can view
     // the full-page screenshot in the image that ends with "-full".
```

Comment:
This comment should probably be updated to explain that the diff screenshot is generated using a different algorithm from the algorithm that we use to grade image similarity, so that people aren't confused if the diffs show a lot of sub-pixel font rendering differences that don't really matter.

Reply:
Good idea.  Done.